manifest.go

```diff
@@ -450,16 +450,13 @@ func fetchManifestEntries(m ManifestFile, fs iceio.IO, discardDeleted bool) ([]M
 	}
 	defer f.Close()
 
-	dec, err := ocf.NewDecoder(f)
+	dec, err := ocf.NewDecoder(f, ocf.WithDecoderSchemaCache(&avro.SchemaCache{}))
 	if err != nil {
 		return nil, err
 	}
 
 	metadata := dec.Metadata()
-	sc, err := avro.ParseBytes(dec.Metadata()["avro.schema"])
-	if err != nil {
-		return nil, err
-	}
```

Comment:
I just happened to notice this and updated it so it doesn't re-parse the schema.